lib/zold/wallets.rb

```diff
@@ -32,11 +32,18 @@ def initialize(dir)
       @dir = dir
     end
 
-    # @todo #70:30min Let's make it smarter. Instead of returning
-    #  the full path let's substract the prefix from it if it's equal
-    #  to the current directory in Dir.pwd.
+    # Returns wallets directory path
     def to_s
-      path
+      dir = Dir.pwd[1..-1].split('/')
```

Comment:
@mulev I don't think this is cross platform. Will fail on Windows, right?

Reply:
@yegor256 on Windows Dir.pwd also uses "/" as a separator, so the split is correct, but the first part is wrong. Now fixed